src/SortableContainer/index.js

```diff
@@ -105,12 +105,18 @@ export default function sortableContainer(WrappedComponent, config = {withRef: f
         useWindowAsScrollContainer,
       } = this.props;
 
+      const findScrollTop = () => {
+        if (this.document.body['scrollTop']) { return this.document.body; }
```

Comment:
how about: 
if('scrollTop' in this.document.body)

Reply:
Also, please note that in Chrome scrollTop is present in both but appears to be writeable ONLY in document.documentElement but not in document.body